ucsmsdk/ucsmo.py

```diff
@@ -227,6 +227,22 @@ def __set_prop(self, name, value, mark_dirty=True, forced=False):
                 self._dirty_mask |= prop_meta.mask
         object.__setattr__(self, name, value)
 
+    def __json__(self):
+        # return the json dict
+        dict = {'class_id': self._class_id}
+        for prop, prop_value in sorted(ucsgenutils.iteritems(self.__dict__)):
+            if prop in ManagedObject.__internal_prop or prop.startswith(
+                    "_ManagedObject__"):
+                continue
+            if prop in self.__xtra_props:
+                del dict[prop]
```

Comment:
yes. You are right! I have deleted this line in my code. But forgot to change it right here. I will update it right now.